.github/BOTMETA.yml

```diff
@@ -623,207 +690,310 @@ files:
   lib/ansible/plugins/connection/network_cli.py:
+    support: network
     maintainers: $team_networking
     labels: networking
   lib/ansible/plugins/connection/persistent.py:
     maintainers: $team_networking
     labels: networking
+  lib/ansible/plugins/connection/winrm.py:
+    maintainers: $team_windows_core
+    labels:
+      - windows
+###############################
+# plugins/filters
+  lib/ansible/plugins/filters:
+    support: core
+  lib/ansible/plugins/filter/network.py:
+    support: network
+    maintainers: $team_networking
+    labels: networking
+###############################
+# plugins/httpapi
+  lib/ansible/plugins/httpapi:
+    support: network
+    maintainers: $team_networking
+    labels: networking
   lib/ansible/plugins/httpapi/ftd.py:
+    support: community
     maintainers: annikulin $team_networking
-    labels: networking
+###############################
+# plugins/inventory
+  lib/ansible/plugins/inventory/__init__.py:
+    support: core
+  lib/ansible/plugins/inventory/ini.py:
+    support: core
   lib/ansible/plugins/inventory/openstack.py:
     maintainers: $team_openstack
     keywords:
-    - openstack
-    - inventory
+      - openstack
+      - inventory
     labels:
-    - cloud
-  lib/ansible/plugins/cliconf/voss.py:
-    maintainers: $team_extreme
-    labels: networking
-  lib/ansible/plugins/connection/winrm.py:
-    maintainers: $team_windows_core
-    labels:
-    - windows
+      - cloud
   lib/ansible/plugins/inventory/ovirt4.py:
     maintainers: machacekondra
     labels:
       - ovirt
       - cloud
   lib/ansible/plugins/inventory/vultr.py: *vultr
   lib/ansible/plugins/inventory/scaleway.py: *scaleway
-  lib/ansible/plugins/filter/network.py:
-    maintainers: $team_networking
-    labels: networking
-  lib/ansible/plugins/lookup/nios.py:
-    maintainers: $team_networking sganesh-infoblox
-    labels:
-      - networking
-      - infoblox
-  lib/ansible/plugins/lookup/nios_next_ip.py:
-    maintainers: $team_networking
-    labels: networking
-  lib/ansible/plugins/lookup/dig:
-    maintainers: jpmens
-    labels: community
+  lib/ansible/plugins/inventory/yaml.py:
+    support: core
+###############################
+# plugins/loader
+  lib/ansible/plugins/loader:
+    support: core
+
+###############################
+# plugins/lookup
+  lib/ansible/plugins/lookup/__init__.py:
+    support: core
   lib/ansible/plugins/lookup/conjur_variable.py:
     maintainers: $team_cyberark_conjur
     notified: cyberark-bizdev
+  lib/ansible/plugins/lookup/dig:
+    maintainers: jpmens
+    labels: community
   lib/ansible/plugins/lookup/cyberarkpassword.py:
     notified: cyberark-bizdev
-  lib/ansible/plugins/netconf/:
-    maintainers: $team_networking
-    labels: networking
+  lib/ansible/plugins/lookup/nios:
+    support: core
+    maintainers: $team_networking sganesh-infoblox
+    labels:
+      - networking
+      - infoblox
   lib/ansible/plugins/lookup/onepassword.py:
```

Comment:
one `lib/ansible/plugins/lookup/onepassword` entry could be used instead of `lib/ansible/plugins/lookup/onepassword.py` and `lib/ansible/plugins/lookup/onepassword_raw.py`